gcloud-java-storage/src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -213,19 +213,20 @@ public Blob update(BlobInfo blobInfo, BlobTargetOption... options) {
   }
 
   /**
-   * Copies this blob to the specified target. Possibly copying also some of the metadata
-   * (e.g. content-type).
+   * Sends a copy request for the current blob to the target blob. Possibly also some of the
+   * metadata are copied (e.g. content-type).
    *
    * @param targetBlob target blob's id
    * @param options source blob options
-   * @return the copied blob
+   * @return a {@link CopyWriter} object that can be used to get information on the newly created
+   *     blob or to complete the copy if more than one RPC request is needed
    * @throws StorageException upon failure
    */
-  public Blob copyTo(BlobId targetBlob, BlobSourceOption... options) {
+  public CopyWriter copyTo(BlobId targetBlob, BlobSourceOption... options) {
     BlobInfo updatedInfo = info.toBuilder().blobId(targetBlob).build();
```

Comment:
I wonder if that is necessary, rather than using `BlobInfo.builder(targetBlob)` when we are not overriding the metadata?